Orange/misc/flagged_data.py

```diff
@@ -0,0 +1,44 @@
+import re
+import numpy as np
+from Orange.data import Table, Domain, DiscreteVariable
+
+FLAGGED_SIGNAL_NAME = "Flagged Data"
```

Comment:
If nothing else, the module belongs within `Orange.widgets` because it includes a name of the signal, hence it **is** related to widgets. :)